recipes/centos_vertx/Dockerfile

```diff
@@ -12,11 +12,10 @@ LABEL che:server:8080:ref=vertx che:server:8080:protocol=http che:server:5005:re
 MAINTAINER Clement Escoffier
 
 ARG VERTX_VERSION=3.4.2
-
 ENV VERTX_GROUPID=io.vertx 
 
-COPY install-vertx-dependencies.sh /tmp/
-RUN sudo chown user:user /tmp/install-vertx-dependencies.sh && \
-    chmod +x /tmp/install-vertx-dependencies.sh && \
-    scl enable rh-maven33 /tmp/install-vertx-dependencies.sh && \
-    sudo rm -f /tmp/install-vertx-dependencies.sh
+ADD configure_maven_and_deps.sh $HOME/
+
+RUN sudo chmod +x ${HOME}/configure_maven_and_deps.sh
+
+CMD ["sh","-c","${HOME}/configure_maven_and_deps.sh"]
```

Comment:
why is it a replaced by a command.
it was a run instruction so it was to cache maven dependencies during the build of the image, not to do it when the container is started ?

Reply:
since we have settings.xml  in place and the `MAVEN_MIRROR_URL` is configurable environment variable, we  need to have entrypoint kind of scripting which can set the MAVEN_MIRROR url as configured by the user.  Also as mentioned earlier comment  I don't see a big value with installed deps during build of images,  as anyways running your build inside spring boot, vert.x or other maven based app in side che will pull it automatically